app/src/main/java/org/mifos/mobile/ui/fragments/UserProfileFragment.kt

```diff
@@ -256,7 +256,7 @@ class UserProfileFragment : BaseFragment(), UserDetailsView {
     override fun showError(message: String?) {
         Toaster.show(rootView, message)
         sweetUIErrorHandler?.showSweetCustomErrorUI(getString(R.string.error_fetching_user_profile),
-                R.drawable.ic_assignment_turned_in_black_24dp, appBarLayout,
+                R.drawable.ic_assignment_not_turned_black_24dp, appBarLayout,
```

Comment:
can we use any existing drawable? I guess it should be present, check other fragments when no internet is present and confirm

Reply:
<img src="https://user-images.githubusercontent.com/55937724/102262162-af7f3500-3f38-11eb-9c41-5e1acb3e0399.png" width="350" height="800">
Does this looks appropriate ?. I have used an existing drawable for this.

Reply:
@miPlodder Please review the updated changes.